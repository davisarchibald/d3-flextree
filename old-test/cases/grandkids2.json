{
  "x_size": 10,
  "y_size": 10,
  "children": [
    {
      "x_size": 10,
      "y_size": 10
    },
    {
      "x_size": 10,
      "y_size": 10,
      "children": [
        { 
          "x_size": 20,
          "y_size": 10
        },
        { 
          "x_size": 20,
          "y_size": 10
        },
        { 
          "x_size": 10,
          "y_size": 10
        },
        { 
          "x_size": 10,
          "y_size": 10
        },
        { 
          "x_size": 10,
          "y_size": 10
        }
      ]
    },
    {
      "x_size": 10,
      "y_size": 10
    }
  ]
}
