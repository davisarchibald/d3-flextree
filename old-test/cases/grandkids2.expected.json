{
  "x" : 0.0,
  "y" : 0.0,
  "children" : [ 
    {
      "x" : -10.0,
      "y" : 10.0
    },
    {
      "x" : 0.0,
      "y" : 10.0,
      "children": [
        {
          "x" : -25.0,
          "y" : 20.0
        },
        {
          "x" : -5.0,
          "y" : 20.0
        },
        {
          "x" : 10.0,
          "y" : 20.0
        },
        {
          "x" : 20.0,
          "y" : 20.0
        },
        {
          "x" : 30.0,
          "y" : 20.0
        }
      ]
    },
    {
      "x" : 10.0,
      "y" : 10.0
    }
  ]
}
